schemas/resources/schemas/data-quality-summary/v1.0.0/index.json

```diff
@@ -5,7 +5,7 @@
     "Version": "0.0.1"
   },
   "Generic": {
-    "Name": "DataQualitySummary",
```

Comment:
so where is this value used, I don't remember? Can we just set this to "Data Quality Summary"? if it is used, don't we have a function in one of the util.clj files to convert names with spaces to - or is that for _?

Reply:
found it: (camel-snake-kebab.core/->kebab-case "this is my text")

Reply:
Done! The 2 files that were changed in order to implement a human-readable "Name" field in the schema (that uses real spaces) are:
`indexer-app/src/cmr/indexer/data/concepts/generic.clj` (line 78)
`ingest-app/src/cmr/ingest/api/generic_documents.clj` (line 60)

Reply:
so resolve this then?

Reply:
With or without spaces?